webpack-widget.js

```diff
@@ -5,38 +5,56 @@ const CleanPlugin       = require('clean-webpack-plugin')
 const CopyPlugin        = require('copy-webpack-plugin')
 const ExtractTextPlugin = require('extract-text-webpack-plugin')
 const ZipPlugin         = require('zip-webpack-plugin')
-
-const MateriaDevServer = require('./express');
-
-// Default Materia Widget Config
-const defaultCfg = {
-	cleanName: '',
-	srcPath: path.join(process.cwd(), 'src'),
-	outputPath: path.join(process.cwd(), 'build'),
-	demoPath: 'demo.json',
-	installPath: 'install.yaml',
-	iconsPath: '_icons',
-	scorePath: '_score/',
-	screenshotsPath: '_screen-shots/',
-	assetsPath: 'assets/',
-	preCopy: []
-}
+const MateriaDevServer  = require('./express');
```

Comment:
Breaking Change: MWDK's `getLegacyWidgetBuildConfig` method no long accepts customization of `srcPath`, `outputPath`, `demoPath`, `installPath`, `iconsPath`, `scorePath`, `screenshotsPath`, `assetsPath`, or `preCopy`

The majority of these were created for flexibility, but they really aren't needed and would cause widgets to very greatly if used, increasing complexity.

demoPath is replaced with a new feature that will automatically load `demo_dev.json` if it exists.

preCopy has been replaced with full access to the copy list.